space2stats_api/src/space2stats/__init__.py

```diff
@@ -1,5 +1,3 @@
 """space2stats."""
 
-from .main import fields, summaries  # noqa
```

Comment:
this was the main issue for the test to fail! having this in the `init` meant that as soon has we were doing `from space2stats import` python was complaining about env settings not set 

To be honest I'm not 100% quite sure why (we were using autouse=True for fixture meaning the fixture should be created before doing anything else 🤷‍♂️ )